project.php

```diff
@@ -492,7 +492,7 @@ class="btn-large grey">
 						<b>Share <?php print $villageName; ?>'s Story</b>
 					</h6>
 					<br>
-					<?php printShareButtons($projectId, $projectName, $projectName, 60); ?>
```

Comment:
What's your thinking with removing these share buttons?

Reply:
didn't mean to remove the share buttons. just wanted to add them to fundraisers.